packages/tags/src/elements/Avatar.tsx

```diff
@@ -5,10 +5,11 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes } from 'react';
+import React from 'react';
 import { StyledAvatar } from '../styled';
+import { IStyledBaseIconProps } from '@zendeskgarden/react-theming';
 
-const AvatarComponent = (props: HTMLAttributes<HTMLElement>) => <StyledAvatar {...props} />;
+const AvatarComponent = (props: IStyledBaseIconProps) => <StyledAvatar {...props} />;
```

Comment:
Let me know if reusing the newly exported `IStyledBaseIconProps` isn't recommended.

Reply:
hmm, I would discourage it for the sake of clarity. This should be `props: SVGAttributes<SVGElement>` and the `@extends` clause updated below accordingly. Also, helps avoid a peer dependency mismatch (even though our ADR expects theming to be up-to-date).